docs/explanation/moderation-and-spam-control.md

```diff
@@ -0,0 +1,81 @@
+# Moderation and spam control
+
+Moderation and spam control are critical to maintaining the health and security
+of any communication platform, and Synapse is no exception. As a Matrix
+homeserver, Synapse enables decentralized communication, which brings challenges
+in combating abuse and ensuring user safety. This explanation
+explores charm configurations and tools (Synapse Invite Checker Module and Mjolnir)
+that can help set moderation and spam control in Synapse.
+
+---
+
+## Charm Configurations
+
+Synapse charm provide administrators with configuration options to fine-tune
+moderation and spam control. Below are some key settings:
+
+#### [`rc_joins_remote_burst_count`](https://charmhub.io/synapse/configurations#rc_joins_remote_burst_count)
```

Comment:
Should this heading (and the subsequent headings in this section) be H3 instead of H4? Unless there was a specific reason for using H4?

Reply:
Good catch, no specific reason, I changed to H3 now.